lib/ui/prompt.js

```diff
@@ -43,7 +43,12 @@ PromptUI.prototype.run = function (questions) {
       return this.answers;
     }.bind(this), {})
     .toPromise(Promise)
-    .then(this.onCompletion.bind(this));
+    .then(this.onCompletion.bind(this))
+    .catch(function (error) {
+      console.error(error);
+
+      process.exit();
```

Comment:
Let's not `process.exit`, this prevents users from handling the exception.

Reply:
Already deleted ```process.exit```.